assets/components/ovn/configmap.yaml

```diff
@@ -0,0 +1,36 @@
+---
+# The ovnconfig config file. Used by both node and master processes.
+kind: ConfigMap
+apiVersion: v1
+metadata:
+  name: ovnkube-config
+  namespace: openshift-ovn-kubernetes
+data:
+  ovnkube.conf:   |-
+    [default]
+    mtu="1400"
+    cluster-subnets="10.128.0.0/14/23"
```

Comment:
Should this come from the microshift config file?

Reply:
fixed to use cluster cidr from microshift.conf.